airflow/providers/amazon/aws/utils/connection_wrapper.py

```diff
@@ -127,13 +139,33 @@ def __post_init__(self, conn: "Connection"):
         init_credentials = self._get_credentials(**extra)
         self.aws_access_key_id, self.aws_secret_access_key, self.aws_session_token = init_credentials
 
-        if not self.region_name and "region_name" in extra:
-            self.region_name = extra["region_name"]
-            self.log.info("Retrieving region_name=%s from %s extra.", self.region_name, self.conn_repr)
+        if not self.region_name:
+            if "region_name" in extra:
+                self.region_name = extra["region_name"]
+                self.log.info("Retrieving region_name=%s from %s extra.", self.region_name, self.conn_repr)
```

Comment:
Should this be debug level?

Reply:
Yeah. I change for a lot of stuff from `DEBUG` to `INFO` severity levels by previous PR's incidentally.
I will revert mos of them to debug.